### DISCUSSION TITLE
How to set the Logback MDC field for searchable tags in SkyWalking's log monitoring?

### DISCUSSION THREAD

**Discussion by poisonduyao** [apache/skywalking#11342] [Q&A]
I want to set up a Mapped Diagnostic Context (MDC) field that can be searched in the log tags, how can I do this? I am currently using MySQL for storage.

**Comment by wu-sheng** [comment]
Follow the tip on the UI

<img width="1450" alt="image" src="https://github.com/apache/skywalking/assets/5441976/2d27cb85-ef9d-48f8-b4d0-c8592a284703">

**Reply by poisonduyao** [reply]
I have already set up this configuration file, but it still doesn't work.

**Reply by wu-sheng** [reply]
I can't guess why. I can see the listed tags are set through this way. So, it should work well.
